.github/workflows/test.yaml

```
@@ -62,10 +64,12 @@ jobs:
     env:
       REQUIRE_ALL_TESTS: 1
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@9c091bb21b7c1c1d1991bb908d89e4e9dddfe3e0 # v7.0.0
+        with:
+          persist-credentials: 'false'
 
       - name: Set up Python ${{ matrix.python-version }}
```

actionlint (1.7.12)

[error] property "python-version" is not defined in object type {platform: string}

(expression)